stock_reserve_rule/migrations/13.0.1.1.0/post-migration.py

```diff
@@ -0,0 +1,18 @@
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html)
+
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+def migrate(cr, version):
+    cr.execute(
+        """
+    INSERT INTO stock_picking_type_stock_reserve_rule_rel
+      (stock_reserve_rule_id, stock_picking_type_id)
+    SELECT id, picking_type_id
```

Comment:
I'm not sure we still have the `picking_type_id` field on `stock_reserve_rule` once the module has been updated.

Reply:
We do. I checked this and run the upgrade on the project's DB. In fact, the column is removed after the migration script is called.

Reply:
Good to know, thanks!